linera-base/src/data_types.rs

```diff
@@ -945,6 +954,32 @@ impl From<BlobContent> for CompressedBytecode {
     }
 }
 
+impl TryFrom<UserApplicationDescription> for BlobContent {
+    type Error = bincode::Error;
+
+    fn try_from(application_description: UserApplicationDescription) -> Result<Self, Self::Error> {
+        Ok(Self {
+            bytes: bincode::serialize(&application_description)?,
```

Comment:
@afck @andresilva91 We may want to consistently use `bcs` for everything storage related